sections/all_map_locations.tex

```diff
@@ -3,18 +3,19 @@
 
 \begin{multicols*}{2}
 
-\hypertarget{All}{This} section describes the function of every single Field in the game.
-Any Fields not shown below or in the above Expansion section are explained by the reference sheet found at the back of the official rule book, pictured on the right. All such fields are \textbf{Visitable}.\par
+\hypertarget{All}{This} section describes the function of Fields not explained above.
+The following \textbf{Visitable} fields are not pictured out of redundancy, as their effects are described clearly by the symbols at the back of the official rule book (pictured on the right): \textbf{Water Wheel}, \textbf{Windmill}, \textbf{Mystical Garden}, \textbf{Shrine of Magic Gesture/Incantation}, \textbf{Learning Stone}, \textbf{Temple}, \textbf{Warrior's Tomb}, \textbf{Fountain of Youth}, \textbf{Treasure Symbol}, \textbf{Artifact Symbol}, \textbf{Resource Symbol}, \textbf{Pandora's Box}.\par
 \bigbreak
-\note{10}{
+\note{8}{
```

Comment:
Why? Sparse is better than condensed. Maybe a `9` at least?

Reply:
It was smaller because of the ballista, which is gone for now.